config/config.yaml

```diff
@@ -1,5 +1,8 @@
 scan_front_topic_name: "/scan"
 scan_back_topic_name: "/scan_rplidar"
-fused_scan_topic_name: "/fused_scan_crop"
-robot_width: 0.6 # in the lateral dirctins
-robot_length: 0.68 # in the direction of the motion
+fused_scan_topic_name: "/fused_scan_temp"
+robot_width: 0.6
+robot_length: 0.65
+base_link: "base_link"
+polygon_x: [0.3, 0.3, -0.3, -0.3]
```

Comment:
change the format, both the coordinate of a single point should be given together.
It should be like [[x1, y1], [x2, y2], [x3, y3]]